pkg/interface/chat/src/js/components/new.js

```diff
@@ -38,9 +41,18 @@ export class NewScreen extends Component {
     }
   }
 
-  idChange(event) {
+  titleChange(event) {
+    let asciiSafe = event.target.value
+      .replace(/[^a-z0-9~_.-]/g, "-");
```

Comment:
This replaces capitals with `-`

Reply:
fine by me for a first pass. if it's easy to make a better one, i'd love to see it. i'm not the best at regex

Reply:
you could just stick a `toLowerCase()` on it before the regex then so we preserve the letters in the path name. This is the path name, right? Not the title itself?

Reply:
Yes, path name must be ascii safe. That’s a good idea